tools/bm-maintenance/deployment_tdx_snp.yml

```diff
@@ -47,7 +47,7 @@ spec:
           serviceAccountName: cleanup-sa
           containers:
             - name: cleanup
-              image: ghcr.io/edgelesssys/contrast/cleanup-bm:v0.0.1
+              image: ghcr.io/edgelesssys/contrast/cleanup-bare-metal:v0.0.1
```

Comment:
How is the image updated after push? How do we ensure the deployment is running with the latest image?

Reply:
I think you'd have to set PullPolicy to always, otherwise it will just use the cached image.

Reply:
If the image tag is `latest` the default is to always pull the image: https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting.